packages/renderer/MdToHtml/rules/fountain.ts

```diff
@@ -107,12 +107,19 @@ const pluginAssets = function() {
 function renderFountainScript(markdownIt: any, content: string) {
 	const result = fountain.parse(content);
 
-	return `
-		<div class="fountain joplin-editable">
-			<pre class="joplin-source" data-joplin-language="fountain" data-joplin-source-open="\`\`\`fountain&#10;" data-joplin-source-close="&#10;\`\`\`&#10;">${markdownIt.utils.escapeHtml(content)}</pre>
+	let optionalTitlePage = '';
```

Comment:
Please name it "titlePageHtml" to indicate that it's HTML content that does not need to be escaped (because I was going to ask you to escape it but indeed it doesn't need to). See [Make wrong code look wrong](https://joplinapp.org/help/dev/coding_style#make-wrong-code-look-wrong)